newdle/client/src/components/MyNewdles.js

```diff
@@ -100,11 +101,19 @@ function MyNewdle({newdle: {code, title, participants, duration, final_dt: final
         <Icon name="users" />
         <label>
           <em>
-            {!participants.length
-              ? 'There are no participants yet'
-              : answers.length !== participants.length
-              ? `${answers.length} out of ${participants.length} participants answered`
-              : 'All participants answered'}
+            {!participants.length ? (
+              t`There are no participants yet.`
+            ) : answers.length !== participants.length ? (
+              <Trans>
+                <Plural
+                  value={participants.length}
+                  one={`${answers.length} out of # participant answered`}
+                  other={`${answers.length} out of # participants answered`}
```

Comment:
I think it's fine now, but if we ever end up a language where the number of people who answered also affects the translation we'd probably need to nest yet another plural inside each of these two options